src/style.css

```
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  margin-top: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form__field {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: box-shadow, 0.3s;
+  outline: none;
+  position: relative;
+  padding: 0 10px;
+}
+
+.form__field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form__field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.form__field::placeholder {
+  color: #3d4e61;
+}
+
+.form__input-big {

```

```suggestion
.form--big .form__input {
```
you don't need another modifier for input